modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/transcripts/TranscriptPage.kt

```diff
@@ -146,6 +153,21 @@ private fun TranscriptContent(
             .fillMaxWidth()
             .background(colors.backgroundColor()),
     ) {
+        val scrollState = rememberScrollState()
+        var viewportHeight by rememberSaveable { mutableFloatStateOf(0f) }
+        val textModifier = Modifier
+            .padding(horizontal = 16.dp)
+            .padding(bottom = bottomPadding)
+            .verticalScroll(scrollState)
+            .verticalScrollBar(
+                thumbColor = colors.textColor(),
+                scrollState = scrollState,
+                viewPortHeight = viewportHeight,
+            )
+            .onGloballyPositioned {
+                viewportHeight = it.boundsInParent().height
+            }
```

Comment:
It would nice to avoid `onGloballyPositioned` here and make it part of `verticalScrollBar()`. I think something like this should work.

```kotlin
fun Modifier.verticalScrollBar() = composed {
    var viewPortHeight by remember { mutableFloatStateOf(0f) }
    drawWithContent {

    }.onGloballyPositioned { viewPortHeight = it.boundsInParent().height }
}
```

Reply:
Moved in [2e93c16](https://github.com/Automattic/pocket-casts-android/pull/2556/commits/2e93c1646151a1c49401f9669adda2f9067cad62)